protocol/common/message_linkpreview.go

```diff
@@ -31,6 +31,7 @@ type LinkPreview struct {
 	Hostname    string                         `json:"hostname"`
 	Title       string                         `json:"title,omitempty"`
 	Description string                         `json:"description,omitempty"`
+	Favicon     string                         `json:"icon,omitempty"`
```

Comment:
Why not `favicon` in json? 😄 
```suggestion
	Favicon     string                         `json:"favicon,omitempty"`
```

Reply:
Done, Can you kindly take another look?